packages/shared/package.json

```diff
@@ -6,6 +6,7 @@
 		"@acala-network/chopsticks-testing": "1.0.2-2",
 		"@e2e-test/networks": "workspace:*",
 		"@polkadot/api": "^15.0",
+		"@polkadot/react-hooks": "^0.39.1",
```

Comment:
I don't think this is needed?

Reply:
Local experiments I forgot to remove; this would be for my next PR.

Removed.